googleapi/googleapi.go

```diff
@@ -417,4 +417,23 @@ type traceTok string
 
 func (t traceTok) Get() (string, string) { return "trace", "token:" + string(t) }
 
+type queryParameter struct {
+	key    string
+	values []string
+}
+
+// QueryParameter allows setting the value(s) of an arbitrary key.
+func QueryParameter(key string, values ...string) CallOption {
+	return queryParameter{key: key, values: append([]string{}, values...)}
+}
+
+// Get will never actually be called -- GetMulti will.
```

Comment:
This is a public package, I think this needs more documentation.

maybe add the type to googleapi:

```
type MultiCallOption interface {
   CallOption
   GetMulti() (string, []string)
}
```

then QueryParameter can return a MultiCallOption (or a better name, not sure)
and then document on MultiCallOption the difference in behavior.

Reply:
Done.